newsroom/news_api/news/assets/assets.py

```diff
@@ -0,0 +1,38 @@
+import superdesk
+import flask
+from newsroom.news_api.settings import URL_PREFIX
+from newsroom.news_api.api_tokens import CompanyTokenAuth
+from flask import abort
+from newsroom.upload import ASSETS_RESOURCE
+from flask_babel import gettext
+import bson.errors
+from werkzeug.wsgi import wrap_file
+from newsroom.news_api.utils import post_api_audit
+
+blueprint = superdesk.Blueprint('assets', __name__)
+
+
+@blueprint.route('/{}/assets/<path:asset_id>'.format(URL_PREFIX), methods=['GET'])
```

Comment:
Instead of setting `URL_PREFIX` here it might be better to add this in the factory method that registers the blueprints: https://github.com/superdesk/newsroom/blob/master/newsroom/factory/app.py#L132

Similar to how it is done in Superdesk (by getting `api_prefix` from the app at runtime):
https://github.com/superdesk/superdesk-core/blob/develop/superdesk/__init__.py#L90

This would also have to be changed for `newsroom.news_api.news.item.item`
https://github.com/superdesk/newsroom/blob/master/newsroom/news_api/news/item/item.py#L13

Reply:
Done